src/wishlist.tm

```diff
@@ -21,7 +21,55 @@
 
   \ <hrule>
 
-  \;
+  Sometimes, one has to produce quite compacts documents, where space is key
+  (e.g.<nbsp>a short report, an exercice sheet or correction<text-dots>). One
+  very important feature you be to put figures on the left or the right of
+  the page, in a floating manner. To emulate this, tables can be used, with
+  one cell for the text and one cell for the figure, but it does not work
+  well at all with figure legends, footnotes<text-dots>
+
+  More generally, mouse-movable floating minipages would be a killer feature
+  and solve many issues at the same time.
+
+  <hrule>
+
+  An user-friendly interface to create/modify shortcuts.
+
+  <hrule>
+
+  Being able to easily create macros, at least non-parametrical ones, with a
```

Comment:
What is difficult in the current implementation? Again, is not clear what is wrong and how a developer could make it simpler.

Reply:
I simply never realized there was already a menu for that.